docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,197 @@
+
+[role="screenshot"]
+image::images/highlights-maps-hexagon.png[region map created in Lens]
+
+[float]
+==== Create region maps from Lens
+
+You can now create region maps
+(also called choropleth maps) from *Lens*.
+
+[role="screenshot"]
+image::images/highlights-maps-region.png[Apply changes to render visualization]
+
+[float]
+==== Add custom icons
+
+Upload an SVG image and use it as icons in your maps.
+Edit your maps layer, and then select
+*Layer Style> Symbol style > Icon > Add custom icon*.
+
+[role="screenshot"]
+image::images/highlights-maps-custom-icon.png[Add custom icon button in layer style settings]
+
+[role="screenshot"]
+image::images/highlights-maps-custom-icon-2.png[custom phone icon added to a map]
+
+[float]
+=== Alerting
+
+[float]
+==== Temporarily pause rules
+
+*Alerting* introduces the ability to snooze rules to suppress notifications
+and actions for a user-defined duration. You'll never miss an
+action because you forgot to unmute a rule while handling problems that arose
+unexpectedly or during known downtimes.
+
+[role="screenshot"]
+image::images/highlights-alerting-snooze.png["UI for enabling snooze mode", width=50%]
+
+[float]
+==== View rule history
+
+For better observability and monitoring of your rules, {kib} now
+displays the execution history of your rules and actions. You can customize the
+history view to include granular information with an array of impactful data points,
+including the underlying alerting query durations,
+the number of actions that were triggered or errored, alert context,
+rule execution timeouts, and more.
+
+[role="screenshot"]
+image::images/highlights-alerts-rule-history.png[Rule history table in Alerting]
+
+[float]
+==== Configure rules
+
+*Alerting* introduces configurable guardrails so that as an admin,
+you can apply your policies on a level deeper than RBAC and fully stay on top of
+the stress that the alerting system may put on your instance, in case your users misconfigured rules.
+This way you have peace of mind that your instance remains performant.
+
+Specifically, in the kibana.yml file:
+
+* Configure your rule queries to timeout after a custom time expires without the query returning.
+You can also do this on a per rule type basis.
+Check the {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the `xpack.alerting.run.timeout` setting.
+
+* Configure a minimum value for the interval with which your rules run. Strictly enforce the
+minimum interval or just display warnings on the UI. Check the
+ {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the
+`xpack.alerting.rules.minimumScheduleInterval` settings.
+
+* Configure the maximum number of actions that a rule can generate. Specify a different cap for your
+different rule types, depending on your needs.
+Check the {kibana-ref}/alert-action-settings-kb.html#action-settings[documentation] for the `xpack.alerting.run.actions.max` setting.
+
```

Comment:
Since all three links above go to the same place, IMO it makes sense to just put the link once at the bottom:
```suggestion
For more information, refer to {kibana-ref}/alert-action-settings-kb.html#action-settings[Action settings].
```